template/client/store/index.js

```diff
@@ -11,67 +13,67 @@ import type {
   RawCommentType
 } from '~/types'
 
-function state (): StateType {
-  return {
-    comments: null
-  }
-}
-
-const getters = {
-  hasComments (state: StateType): boolean {
-    return Boolean(state.comments && state.comments.length > 0)
-  }
-}
+Vue.use(Vuex)
 
-const mutations = {
-  [mutationTypes.SET_COMMENTS]: (
-    state: StateType, comments: Array<RawCommentType>
-  ) => {
-    const updatedComments: Array<CommentType> = []
-
-    for (const comment of comments.slice(0, 10)) {
-      const newOne = {
-        ...comment,
-        rating: 0 // try to comment out this line
-      }
-      updatedComments.push(newOne)
-    }
+const createStore = () => {
+  return new Vuex.Store({
```

Comment:
Can you please refactor this function like so:

```js
return new Vuex.Store({
   state, 
   mutations,
   getters,
   actions
})
```

And create separate `store/actions.js`, `store/mutations.js`, `store/state.js`, `store/getters.js` files?

This way it would be way more readable.
And that's how large projects are defined anyways.

Sorry, I forgot to mention it earlier.